src/qml/filters/audio_lowpass/ui.qml

```diff
@@ -21,31 +21,52 @@ import QtQuick.Controls 2.12
 import QtQuick.Layouts 1.12
 import Shotcut.Controls 1.0 as Shotcut
 
-Item {
+Shotcut.KeyframableFilter {
+    property string cutoffProperty: '0'
+    property string stagesProperty: '1'
+    property string wetnessProperty: 'wetness'
     width: 350
     height: 125
+    keyframableParameters: preset.parameters
+    startValues: [0, 0, 0]
+    middleValues: [0, 0, 1]
+    endValues: [0, 0, 0]
```

Comment:
The initial values for the non-wetness params in these arrays should be the same as the defaults below and thus:
```
     startValues: [2637, 1, 0]
     middleValues: [2637, 1, 1]
     endValues: [2637, 1, 0]
```